app.py

```diff
@@ -129,8 +129,11 @@ def in_container():
     parser.add_argument("--host", default="127.0.0.1")
     parser.add_argument("--port", default=20124, type=int)
     parser.add_argument("--reload", action="store_true")
+    parser.add_argument("--url", default="https://conda.anaconda.org/")
     args = parser.parse_args()
 
+    URL = args.url.rstrip('/')
```

Comment:
variable names should be `lower_case_with_underscores`

Reply:
I believe this one counts as a global module variable and I use it as a global in line 31. I did not see a way to explicitly pass it `fetch_artifact_graph` (though I may be mistaken) and I felt it was less confusing to use global variable convention. I am still happy to lowercase it if that is preferred.

Reply:
Yeah, it is inconsistent with parser, args, etc as it stands.  Really everything here should be in a main() function, but I'm not asking you to rearrange here :wink: